src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -393,37 +389,23 @@ namespace winrt::TerminalApp::implementation
             {
                 _filteredActions.Append(action);
             }
-        }
 
-        switch (_currentMode)
-        {
-        case CommandPaletteMode::ActionMode:
-            SearchBoxText(RS_(L"CommandPalette_SearchBox/PlaceholderText"));
-            NoMatchesText(RS_(L"CommandPalette_NoMatchesText/Text"));
-            ControlName(L"Command Palette");
-            break;
-        case CommandPaletteMode::TabSwitcherMode:
-        {
-            SearchBoxText(RS_(L"TabSwitcher_SearchBoxText"));
-            NoMatchesText(RS_(L"TabSwitcher_NoMatchesText"));
-
-            if (_anchorKey != VirtualKey::None)
+            switch (_currentMode)
             {
-                _searchBox().Visibility(Visibility::Collapsed);
-                ControlName(L"Tab Switcher Anchored Mode");
-            }
-            else
+            case CommandPaletteMode::TabSwitcherMode:
             {
-                ControlName(L"Tab Switcher UnAnchored Mode");
+                SearchBoxText(RS_(L"TabSwitcher_SearchBoxText"));
+                NoMatchesText(RS_(L"TabSwitcher_NoMatchesText"));
+                ControlName(RS_(L"TabSwitcherControlName"));
+                break;
+            }
+            // Let's say the default mode for now is ActionMode.
+            default:
```

Comment:
```suggestion
            case CommandPaletteMode::ActionMode:
            default:
```

we can leverage case fallthrough to _make sure ActionMode is represented_ but also give it the behavior of default

Reply:
it's better to always make sure the switch-case covers all the enum values!